src/test/regress/multi_schedule

```diff
@@ -96,6 +96,11 @@ test: tableam
 test: propagate_statistics
 test: pg13_propagate_statistics
 
+# ----------
+# Test for alter index stmts
+# ----------
+test: alter_index
```

Comment:
could probably run in parallel with something else (move it to a line with existing tests)